OpenEdXMobile/src/main/java/org/edx/mobile/module/db/impl/IDatabaseImpl.java

```diff
@@ -757,26 +757,14 @@ public Boolean isAnyVideoDownloadingInSubSection(final DataCallback<Boolean> cal
     }
 
     @Override
-    public List<String> getUniqueCourseIdsForDownloadedVideos(final DataCallback<List<String>> callback) {
+    public List<String> getUniqueCourseIdsForDownloadedVideos(@Nullable final DataCallback<List<String>> callback) {
```

Comment:
Since, I've just touched this method in my PR, so, adding the `@Nullable` in the interface declaration as well.